plugins/src/lib.rs

```diff
@@ -289,6 +289,23 @@ pub fn service(attr: TokenStream, input: TokenStream) -> TokenStream {
         vis,
         args,
         method_attrs: &rpcs.iter().map(|rpc| &*rpc.attrs).collect::<Vec<_>>(),
+        method_cfgs: &rpcs
+            .iter()
+            .map(|rpc| {
+                rpc.attrs
+                    .iter()
+                    .filter(|att| {
+                        att.style == AttrStyle::Outer
+                            && match &att.meta {
+                                syn::Meta::List(syn::MetaList { path, .. }) => {
+                                    path.get_ident() == Some(&Ident::new("cfg", rpc.ident.span()))
+                                }
+                                _ => false,
+                            }
+                    })
+                    .collect::<Vec<_>>()
```

Comment:
nit: since this might not be super obvious to a reader, perhaps move it into a function called something like `CollectCfgAttrs`?

Reply:
Done with 8d9e433e908bdde54d5e781f5af69aad02d28e07